api/src/controllers/auth.controller.ts

```diff
@@ -38,18 +40,45 @@ export class AuthController {
     private userService: UserService,
     private jwtService: JwtService,
     private authService: AuthorizationService,
+    @InjectRepository(Invite) private inviteRepo: Repository<Invite>,
+    @InjectRepository(ProjectUser) private projectUserRepo: Repository<ProjectUser>,
     @InjectRepository(ProjectClient) private projectClientRepo: Repository<ProjectClient>,
   ) {}
 
   @Post('signup')
   @HttpCode(HttpStatus.OK)
   async signup(@Body() payload: SignupRequest) {
     if (!config.signupsEnabled) {
-      throw new ForbiddenException('Signups are disabled');
+      let invitesCount = await this.inviteRepo.count({
+        where: { email: payload.email, status: InviteStatus.Sent },
```

Comment:
You gotta normalize the incoming email to make sure it'll look the same as the one in the DB.

Reply:
Moved the validation to a different file.